interfaces/nfs_share/v0/schema.py

```diff
@@ -0,0 +1,47 @@
+"""This file defines the schemas for the provider and require sides of the nfs-share interface.
+
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {
+                endpoint: "nfs://127.0.0.1/data"
```

Comment:
Missing quotes on endpoint
```suggestion
                "endpoint": "nfs://127.0.0.1/data"
```

Reply:
Good catch!